.gitlab-ci.yml

```diff
@@ -164,37 +164,35 @@ Base:
   parallel:
     matrix:
       - RUNNER:
-          - aws/fedora-34-x86_64
-          - aws/fedora-34-aarch64
           - aws/fedora-35-x86_64
           - aws/fedora-35-aarch64
           - aws/rhel-8.5-ga-x86_64
           - aws/rhel-8.5-ga-aarch64
           - aws/centos-stream-8-x86_64
           - aws/centos-stream-8-aarch64
       - RUNNER:
-          - aws/rhel-8.6-nightly-x86_64
-          - aws/rhel-8.6-nightly-aarch64
-          - aws/rhel-9.0-nightly-x86_64
-          - aws/rhel-9.0-nightly-aarch64
+          #- aws/rhel-8.6-nightly-x86_64
+          #- aws/rhel-8.6-nightly-aarch64
+          #- aws/rhel-9.0-nightly-x86_64
+          #- aws/rhel-9.0-nightly-aarch64
           - aws/centos-stream-9-x86_64
           - aws/centos-stream-9-aarch64
         INTERNAL_NETWORK: ["true"]
 
-Manifests:
-  stage: test
-  extends: .terraform
-  rules:
-    - !reference [.upstream_rules, rules]
-    - !reference [.nightly_rules, rules]
-  script:
-    - schutzbot/deploy.sh
-    - /usr/libexec/tests/osbuild-composer/manifest_tests.sh
-  parallel:
-    matrix:
-      - RUNNER:
-          - aws/rhel-9.0-nightly-x86_64
-        INTERNAL_NETWORK: ["true"]
+#Manifests:
+#  stage: test
+#  extends: .terraform
+#  rules:
+#    - !reference [.upstream_rules, rules]
+#    - !reference [.nightly_rules, rules]
+#  script:
+#    - schutzbot/deploy.sh
+#    - /usr/libexec/tests/osbuild-composer/manifest_tests.sh
+#  parallel:
+#    matrix:
+#      - RUNNER:
+#          - aws/rhel-9.0-nightly-x86_64
+#        INTERNAL_NETWORK: ["true"]
```

Comment:
@atodorov you shouldn't have disabled this test, but rather switch it to use the RHEL-9.0 GA runner.

The job does not care about the runner, we just used the latest RHEL version that was available at the time.

Please fix it and re-enable it.